rusk-wallet/src/lib/config.rs

```diff
@@ -0,0 +1,243 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use std::{env, fs, path::PathBuf};
+
+use crate::{LocalStore, WalletArgs};
+
+/// Default Rusk IP address
+pub(crate) const RUSK_ADDR: &str = "127.0.0.1";
+/// Default Rusk TCP port
+pub(crate) const RUSK_PORT: &str = "8585";
+/// Default UDS path that Rusk GRPC-server will connect to
+pub(crate) const RUSK_SOCKET: &str = "/tmp/rusk_listener";
+
+mod parser {
```

Comment:
Not sure why create a new module here. Should either split into a new file, or use the same module.

Reply:
If you look at the content, it's pretty much a clone of the `WalletConfig` with everything optional. This was done because of a limitation of the `toml` crate, and to prevent us from having `unwrap`s throughout the whole program. The content of the `parser` module should not be available to the rest of the crate. This was an attempt to isolate it completely and not have a bunch of utility structs floating around on the same module.